wimprates.py

```diff
@@ -13,56 +13,134 @@
 THIS_DIR = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
 def data_file(path):
     """Convert path in wimprates' data directory to absolute path"""
-    return os.path.join(THIS_DIR, 'data', path) 
+    return os.path.join(THIS_DIR, 'data', path)
 
 ##
 # Halo model
 ##
 
 # Local dark matter density
-rho_dm = 0.3 * nu.GeV/nu.c0**2 /nu.cm**3 
+rho_dm = 0.3 * nu.GeV/nu.c0**2 /nu.cm**3
 
 # Most common velocity of WIMPs in the halo, relative to galactic center (asymptotic)
-v_0 = 220 * nu.km/nu.s       
+v_0 = 220 * nu.km/nu.s
 
 # Velocity of earth/sun relative to gal. center (eccentric orbit, so not equal to v_0)
-v_earth = 232 * nu.km/nu.s   
+v_earth = 232 * nu.km/nu.s
+
+# Mean orbital velocity of the Earth (Lewin & Smith appendix B)
+v_orbit = 29.79 * nu.km / nu.s
 
 # Galactic escape velocity
-v_esc = 544 * nu.km/nu.s     
+v_esc = 544 * nu.km/nu.s
 
 # Maximum dark matter velocity observable on earth
 v_max = v_esc + v_earth
```

Comment:
This is now no longer used. I guess you could remove it and rename v_max_t to just v_max?

Reply:
Thanks, I've renamed v_max_t and removed the redundant line.